drivers/net/ppp.c

```diff
@@ -576,6 +576,20 @@ static int ppp_send(const struct device *dev, struct net_pkt *pkt)
 			protocol = htons(PPP_IP);
```

Comment:
In commit `net: socket: packet: using pckt sckt for passing the PPP dialup data`, remove extra spaces after `Zephyr` string